tests/test_dispatcher.py

```diff
@@ -236,6 +236,23 @@ def test_groups(self, dp):
         sleep(.1)
         assert self.count == 3
 
+    def test_callbackquery_answered_when_no_handler(self, dp, bot, monkeypatch):
+        update = Update(1, callback_query=CallbackQuery(1, User(1, "pool", False), 1, bot=bot))
+        wuhu = []
```

Comment:
please use serious, descriptive, variable names. Even in tests

Reply:
yeah ok